runtime/autoload/provider/clipboard.vim

```diff
@@ -139,7 +139,12 @@ function! provider#clipboard#Executable() abort
     let s:paste['*'] = s:paste['+']
     return 'termux-clipboard'
   elseif !empty($TMUX) && executable('tmux')
-    let s:copy['+'] = ['tmux', 'load-buffer', '-']
+    let [major, minor] = matchlist(systemlist(['tmux', '-V'])[0], 'tmux \(\d\+\)\.\(\d\+\)')[1:2]
+    if major > 3 || (major == 3 && minor >= 2)
+      let s:copy['+'] = ['tmux', 'load-buffer', '-w', '-']
```

Comment:
if tmux 3.2 updates the system clipboard, that potentially avoids pain of xclip, xsel, etc, right? Maybe we should actually prefer tmux as the clipboard backend if tmux 3.2+ is detected?

Reply:
Possibly, but this functionality also depends on the terminal emulator’s support for the OSC 52 escape sequence. xsel and friends don’t, so those are likely still more reliable if the user has an active X connection.